sklearn/ensemble/voting.py

```diff
@@ -454,6 +466,7 @@ def fit(self, X, y, sample_weight=None):
         -------
         self : object
         """
+        y = column_or_1d(y, warn=True)
```

Comment:
I was wondering if we are supporting or not `multioutput` for `VotingRegressor`.

Reply:
the doc says: `y : array-like, shape (n_samples,)` so I guess not

Reply:
OK so we don't have to handle 2D y array then.